validation/src/main/resources/application.conf

```diff
@@ -17,12 +17,20 @@ hmda.validation.quality {
     min-income-for-high-loan=9
     min-income-for-high-loan=${?Q024_MIN_INCOME}
   }
+  Q025 {
+    loan.amount=10
+    loan.amount=${?Q025_LOAN_AMOUNT}
+  }
   Q036 {
     loan.amount=150
     loan.amount=${?Q036_LOAN_AMOUNT}
   }
   Q037 {
-    loan.amount=250
-    loan.amount=${?Q037_LOAN_AMOUNT}
+    loan.amount = 250
+    loan.amount = ${?Q037_LOAN_AMOUNT}
+  }
+  Q038 {
+    loan.amount = 100
+    loan.amount = ${?Q035_LOAN_AMOUNT}
```

Comment:
I notice some back-and-forth about how to format this file. We might want to pick a standard way, and stick with it. (I don't have an opinion of which way.)